CareKitStore/CareKitStore/Structs/OCKOutcomeValue.swift

```diff
@@ -65,10 +65,16 @@ public struct OCKOutcomeValue: Codable, Equatable, CustomStringConvertible {
 
     /// The units for this measurement.
     public var units: String?
-
-    /// The date that this value was created.
+    
+    // The date that this value was created.
```

Comment:
I think you removed the documentation slash by accident, can you add it back?

Reply:
Added